### PR TITLE
fix/grants-3977

### PR DESCRIPTION
**Pull request by Hozan94**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build:watch": "webpack --watch",
     "lint": "standard",
     "lint-fix": "standard --fix",
-    "test": "jest --runInBand",
+    "test": "jest test/integration/narrow/routes/products-processed.test.js --runInBand",
```
Comment (zavies97) [inline package.json:12]: undo this change pls :D

Comment (zavies97) [reply package.json:12]: Also, need a version :')

Comment (Hozan94) [reply package.json:12]: ooopsssssyyyyy

Comment (Hozan94) [reply package.json]: done

**Review by zavies97 — APPROVED** [review]

**Review by zavies97 — APPROVED** [review]
